assignments/DOM Basics/03/index.html

```diff
@@ -35,9 +35,22 @@ <h2>Types of Sharks</h2>
   </body>
 
   <script>
-    const h1 = document.getElementsByTagName("h1")[0];
-    const p = document.getElementsByTagName("p")[0];
-    const ul = document.getElementsByTagName("ul")[0];
     // Your code goes here
+   var doc = document
+   var doc1 = document.ElementChild;
+   var doc2 = document.head;
+   var doc3 = document.title;
+   var doc4 = document.body;
+   var list = document.querySelector("ul");
+   list.firstElementChild.style.backgroundColor = "red";
+   var lis = document.querySelector("ul");
+   lis.querySelectorAll("li");
+   lis.style.backgroundColor = "hotpink";
```

Comment:
It will not work. You will have to use `forEach` for any other way to loo.

```js
var allLis = lis.querySelectorAll("li");
   allLis.forEach(elm => elm.style.backgroundColor = "hotpink");
```